content/docs/0.8.x/manage/shipyard/index.md

```diff
@@ -1,29 +1,306 @@
+## Declare Shipyard (before creating a project)
 
-The most convenient way to change your shipyard is by directly adapting it in the Git *upstream* repository. 
+* A shipyard is defined at the level of a project. This means that all services in a project share the same shipyard definition.
 
-* If you do not have a Git upstream set for your project, please set one since it is the recommended way of using Keptn: [configure Git upstream](../../manage/git_upstream/#create-keptn-project-or-set-git-upstream).
+* A shipyard defines the stages each deployment has to go through until it is released in the final stage, e.g., the production stage.
 
-* If you do not have a Git repository to set an *upstream*, you can update the shipyard using: 
+* A shipyard can consist of any number of stages; but at least one. A stage must have at least the name property.
 
-  ```
-  keptn add-resource --project=PROJECT --resource=./shipyard.yaml --resourceUri=./shipyard.yaml
-  ```
+* A stage can consist of any number of sequences; but at least one.
 
-:warning: Only modify your shipyard in ways mentioned in this section. 
 
-:warning: Make sure to have no running sequence while you are updating the shipyard. Otherwise, running sequences will be updated.
+### Definition of Stage
 
+A stage is declared by its name. This name will be used for the branch in the Git repository and Kubernetes namespace to which services at this stage will be deployed to.
 
-## Add/Remove a task to/from a task sequence
+**Example of a shipyard with three stages:**
+
+```yaml
+apiVersion: spec.keptn.sh/0.2.0
+kind: "Shipyard"
+metadata:
+  name: "shipyard-sockshop"
+spec:
+  stages:
+    - name: "dev"
+    - name: "hardening"
+    - name: "production"
+```
+
+### Definition of Sequence in a Stage
+
+After defining stages, sequences can be added to a stage. A sequence is an ordered list of tasks that are triggered sequentially. A `sequence` consists of the following properties:
+
+* `name`: A unique name of the sequence
+* `on` (optional): An array of events that trigger the sequence.
+* `tasks`: An array of tasks executed by the sequence in the declared order.
+
+**Example:** Extended shipyard with a delivery sequence in all three stage:
+
+```yaml
+apiVersion: spec.keptn.sh/0.2.0
+kind: "Shipyard"
+metadata:
+  name: "shipyard-sockshop"
+spec:
+  stages:
+    - name: "dev"
+      sequences:
+      - name: "delivery"
+        tasks: 
+        - name: "deployment"
+        - name: "release"
+    - name: "hardening"
+      sequences:
+      - name: "delivery"
+        triggeredOn:
+          - event: "dev.delivery.finished"
+        tasks: 
+        - name: "deployment"
+        - name: "release"
+    - name: "production"
+      sequences:
+      - name: "delivery"
+        triggeredOn:
+          - event: "hardening.delivery.finished"
+        tasks: 
+        - name: "deployment"
+        - name: "release"
+```
+
+### Reserved Keptn Tasks
```

Comment:
I like it that you moved this section to this page.

Reply:
We still need to make sure we update this section regularly. It gets outdated so quickly...